docs/source/examples/spot-jobs.rst

```diff
@@ -175,3 +202,17 @@ Here are some commands for managed spot jobs. Check :code:`sky spot --help` for
     # Cancel a spot job by name
     $ sky spot cancel -n bert-qa
 
+
+Advanced note (spot controller)
+-------------------------------
+
+There will be a single spot controller cluster (a small on-demand CPU VM) running in the background to manage all the spot jobs.
+It will be autostopped after 30 minutes of inactivity. Normally, **no user intervention** is needed. 
```

Comment:
> It will be autostopped after 30 minutes of inactivity.

(for clarification on inactivity)

If all spot jobs are finished and no job submission for 30 minutes, the controller will be auto-stopped. Typically no user intervention is needed.

Reply:
Good point! Changed. Thanks!